semgrep/semgrep/commands/ci.py

```diff
@@ -164,6 +164,14 @@ def fix_head_if_github_action(metadata: GitMeta) -> Iterator[None]:
     """,
     envvar="SEMGREP_RULES",
 )
+@click.option(
+    "--dryrun",
```

Comment:
```suggestion
    "--dry-run",
    "dryrun",
```

this would keep consistent with
- `kubectl apply --dry-run`
- `npm install --dry-run`

I added `"dryrun"` just so you can apply the suggestion without changing the variable name below.